packages/dbus.rb

```diff
@@ -36,4 +39,20 @@ def self.build
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
   end
+
+  def self.postinstall
+    FileUtils.mkdir_p "#{CREW_PREFIX}/etc/env.d/"
+    @dbusconfigenv = <<~DBUSCONFIGEOF
+      # Dbus settings
+      DBUS_SYSTEM_BUS_ADDRESS='unix:path=/var/run/dbus/system_bus_socket'
+      [[ "\$DBUS_SESSION_BUS_ADDRESS"=="disabled:" ]] && unset DBUS_SESSION_BUS_ADDRESS
+      if [ -z "\${DBUS_SESSION_BUS_ADDRESS}" ]; then
+        # if not found, launch a new one
+        eval `dbus-launch --sh-syntax`
+        # echo "D-Bus per-session daemon address is: \$DBUS_SESSION_BUS_ADDRESS"
+      fi
+      dbus-update-activation-environment --all
+    DBUSCONFIGEOF
+    IO.write("#{CREW_PREFIX}/etc/env.d/dbus", @dbusconfigenv)
+  end
```

Comment:
This is definitely better to put in install rather than postinstall.  Of course, that would require new binaries.

Reply:
Let's do that then.

Reply:
Could you rebuild for i686?

Reply:
Actually, this needs work. I get a `libaudit.h` not found error on x86_64... Might have to turn off selinux, and maybe we should tweak the runstatedir setting too...

Reply:
Compilation fixed. If you can get an i686 build done @uberhacker. I just need to get my armv7l server connected again before I can do that build.

Reply:
(or not... if none of the packages which depend upon dbus work on i686... maybe it doesn't matter on there?)